corehq/apps/app_manager/static/app_manager/js/download_index_main.js

```diff
@@ -0,0 +1,42 @@
+hqDefine('app_manager/js/download_index_main',[
+    'jquery',
+    'underscore',
+    'ace-builds/src-min-noconflict/ace',
+    'app_manager/js/download_async_modal',
+    'app_manager/js/source_files',
+],function ($, _, ace) {
+
+    $(function () {
+        //Use hqRequire to load the ace modes from CDN
+        hqRequire([
```

Comment:
This doesn't need to go inside the document ready, right? So it could be
```
hqRequire([...], function() {
   $(function () {
      ...
   });
})
```
and potentially start loading the ace modes before the document is done loading.